scripts/create_release_pr

```diff
@@ -111,8 +111,10 @@ else
   sed -i "s/^version:.*/version: $NEXT_CHART_VER/g" "$PELORUS_CHART"
 fi
 
-# Update branch in the Install documentation
-sed -i "s/--branch v.* /--branch v$V_MAJOR.$V_MINOR.$V_PATCH /g" "$INSTALL_DOC"
+# Update branch in the Development documentation
+sed -i "s/\(release number, for example \`\).*\(\`\.\)/\1v$V_MAJOR.$V_MINOR.$V_PATCH\2/g" "$INSTALL_DOC"
```

Comment:
can we add a flag or something to check if the operation was successful?

Reply:
No, sed does not have control over its exit code. We could in theory check by using additional grep or there is hack to "echo" sort of fake exit code directly from sed, however I don't think it's any use here.

We could always check by running `git diff` or `git status` and see if files were changed, but that's really over-design.

Reply:
ok